src/ctelnet.cpp

```diff
@@ -713,10 +713,10 @@ void cTelnet::processTelnetCommand(const string& command)
 
                     if (mpHost->mDiscordAccessFlags & Host::DiscordServerCanSetPresenceId) {
                         infoMessage = tr("[ INFO ]  - Informing Game Server (via GMCP) of Discord Username and Rich "
-                                                     "Presence Guild, Server is allowed to modify Guild to match Game.");
+                                                     "Presence Server is allowed to modify Server to match Game.");
```

Comment:
This is one of those cases where using *Server* for both the Mud Game Server and Discord Server is unhelpful - at the first *Guild* the *Server* you have replaced it with is a different *Server* to the "Server" that follows after the deleted comma - with the result that that sentence *doesn't* make sense - you have to include additional words to disambiguate between the Servers, i.e.:
```
    infoMessage = tr("[ INFO ]  - Informing Game Server (via GMCP) of Discord Username and Rich\n"
                                 "Presence Server in use, the Game Server is allowed to change\n"
                                 "the Rich Presence Server to be one operated by the Game\n"
                                 "Server.");
```

Of course I noticed when I was doing that, that I have forgotten to put in the needed line-feeds at the end of each line on all (but the last line) of all these messages - without them they will just be a single line and disappear off the right of the main console... :sob:

Reply:
I am not even sure if I will be keeping that message - it is not relevant for the user, only for the developer debugging Discord code specifically.